demo-site/shared_config.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
I think having this is unnecessary - or at least it's unnecessary to have the "enforcer_config" property here since the px_config.json file has the same info. I'd say choose one place to put this information - one source of truth.